File: Java/Python/1012-Area.py
line=input().split(" ")
a, b, c = line
pi = 3.14159
tri=(float(a) * float(c))/2
cir=pi*(float(c)*float(c))
tra=float(c)*(float(a)+float(b))/2
qua=float(b)*float(b)
ret=float(a)*float(b)
print('TRIANGULO: {:.3f}'.format(tri))
print('CIRCULO: {:.3f}'.format(cir))
print('TRAPEZIO: {:.3f}'.format(tra))
print('QUADRADO: {:.3f}'.format(qua))
print('RETANGULO: {:.3f}'.format(ret))

File: Java/JavaScript/1012-Area.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split(" ");
var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());
var pi = 3.14159
tri = (a*c)/2;
cir = pi*(c*c);
tra = c*(a+b)/2;
qua = b*b;
ret = a*b;
console.log(`TRIANGULO: ${tri.toFixed(3)}`);
console.log(`CIRCULO: ${cir.toFixed(3)}`);
console.log(`TRAPEZIO: ${tra.toFixed(3)}`);
console.log(`QUADRADO: ${qua.toFixed(3)}`);
console.log(`RETANGULO: ${ret.toFixed(3)}`);